java/core/src/test/java/sleeper/core/statestore/inmemory/InMemoryFileInfoStore.java

```diff
@@ -35,9 +36,19 @@
 import static sleeper.core.statestore.FileInfo.FileStatus.READY_FOR_GARBAGE_COLLECTION;
 
 public class InMemoryFileInfoStore implements FileInfoStore {
-
+    private static final String DEFAULT_TABLE_ID = "test-table-id";
     private final Map<String, PartitionFiles> partitionById = new LinkedHashMap<>();
+    private final Map<String, FileReferenceCount> fileReferenceCounts = new LinkedHashMap<>();
     private Clock clock = Clock.systemUTC();
+    private final String tableId;
+
+    public InMemoryFileInfoStore() {
+        this(DEFAULT_TABLE_ID);
```

Comment:
This would break any test that cared about the table ID on a file reference count matching the TableProperties it's using.